tests/functional/pv/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -98,10 +98,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
             )
             pod_obj.get_fio_results()
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = used_size_before_io + (4 * self.pool_replica)
```

Comment:
@keemano why are you changing from `(16 * self.pool_replica)` to `(4 * self.pool_replica)`

Reply:
> @keemano why are you changing from `(16 * self.pool_replica)` to `(4 * self.pool_replica)`

Previously we were calculating the size after completing IOs in all the 4 files. Now we calculate the size after IO completion in each file.